README.md

```diff
@@ -18,14 +18,21 @@ Conda instructions are provided in more detail, but you may also use `pip` and `
 
 ### Install with pip
 
-To use `pip` to install TorchServe and the model archiver:
+1. Install Java 11
 
-``` bash
-pip install torch torchtext torchvision sentencepiece psutil future
-pip install torchserve torch-model-archiver
-```
+    ```bash
+    sudo apt-get install openjdk-11-jdk
+    ```
+
+1. Use `pip` to install TorchServe and the model archiver:
+
+    ``` bash
+    pip install torch torchtext torchvision sentencepiece psutil future
```

Comment:
Can't we get rid of this line for basic user installation? People don't need any of these, especially sentencepiece... they can download that later as part of the example's setup steps.
In my recent tests, I can get the needed deps just by doing `pip install torchserve torch-model-archiver`.